doc/release-notes.md

```diff
@@ -102,6 +102,9 @@ will trigger BIP 125 (replace-by-fee) opt-in. (#11413)
   option `-deprecatedrpc=banscore` is used. The `banscore` field will be fully
   removed in the next major release. (#19469)
 
+- The `testmempoolaccept` RPC returns `vsize` and a `fee` object with the `base` fee
+  if the transaction passes validation. (#19093)
```

Comment:
I should update this now that it'd be a different PR number. Will incorporate with any review comments so feel free to nit :)

Reply:
updated